app/controllers/secure_controller.js

```diff
@@ -47,7 +50,7 @@ exports.new = async function (req, res) {
         'the frontend state cookie not existing, or the frontend state cookie containing an invalid value.')
       return responseRouter.response(req, res, 'UNAUTHORISED')
     }
-    logging.systemError('Secure controller token', correlationId, chargeTokenId)
+    logging.systemError('Secure controller token', correlationId, chargeId, gatewayAccountId, gatewayAccountType)
```

Comment:
Is `chargeTokenId` the same as `chargeId` or is the change from one to another intentional?
(given the `logSystemError` function it looks like it should have been `chargeId` from the start)

Reply:
`chargeTokenId` is different from `chargeId`. yup, it should have been `chargeId` from the start, which this PR fixes.

Reply:
That's what I though (but wanted to confirm). 👍